src/js/modules/core/components/Masthead.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import injectSheet from 'react-jss';
+import { Link } from 'react-router-dom';
+
+/*
+Site structure navigation links to the first layer items of the site
+(the departments, “subscribe”, “more”). On non-article pages (homepage, department pages),
+ the site structure navigation is displayed in a horizontal bar with a “The Spectator”
+ heading above it (the Header). On article pages, no such navigation exists; there is
+ only direct navigation. On mobile, no such navigation exists; there is only direct navigation.
+ */
+
+const styles = {
+  Masthead: {
```

Comment:
I always name the class of the outer div of JSX in a component the name of the component.